transaction-relayer/src/main.rs

```diff
@@ -338,6 +348,21 @@ fn main() {
         &exit,
     );
 
+    let staked_nodes_overrides = match args.staked_nodes_overrides {
+        None => StakedNodesOverrides::default(),
+        Some(p) => {
+            let file = fs::File::open(&p).expect(&format!(
+                "Failed to open staked nodes overrides file: {:?}",
+                &p
+            ));
+            serde_yaml::from_reader(file).expect(&format!(
+                "Failed to read staked nodes overrides file: {:?}",
+                &p,
+            ))
+        }
+    };
+    let staked_nodes_overrides = Arc::new(RwLock::new(staked_nodes_overrides.staked_map_id));
```

Comment:
doesn't need to be rwlock, that makes sense in validator since admin service needs to modify it

Reply:
oh right

Reply:
fixed